staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -163,6 +170,7 @@ func NewCmdDebug(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobra.
 	}
 
 	addDebugFlags(cmd, o)
+	cmdutil.AddFilenameOptionFlags(cmd, &o.FilenameOptions, "identifying the resource to debug")
```

Comment:
`kubectl exec` only supports `--filename`. Do you think there's a reason we should support kustomize, recursive, etc with kubectl debug?

Reply:
Please correct me if I'm wrong,  but I think, `exec` can also support kustomize, recurive, etc.?.

Reply:
I asked in sig-cli https://kubernetes.slack.com/archives/C2GL57FJ4/p1658931680284329